mods/storage/src/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const constants = {
+  HIGH_WATER_MARK: 1 * 1024,
+  APP_BUCKET: "apps",
+  RECORDINGS_BUCKET: "recordings",
+  PUBLIC_BUCKET: "public",
+  REJECT_UPLOAD_DIRECTORY: "Uploading a directory is not supported"
```

Comment:
This is a simple log comment. It should follow the same pattern we use elsewhere(not on a constant)